src/components/ListOfCities.js

```diff
@@ -0,0 +1,97 @@
+import DisplayInfo from './DisplayInfo';
+import CitySelection from './CitySelection';
+
+//1. import the useState and useEffect hooks from the react library
+import {useState} from 'react';
+
+
+
+//2. import axios
+
+import axios from 'axios';
+
+
+const ListOfCities = () => {
+
+    //3. initialize state to hold the data - cityName will be returned from the api
+    const [cityName, setCityName] = useState("");
+    const [cityKey, setCityKey] = useState("");
+    const [weatherInfo, setWeatherInfo] = useState({});
+   
+
+    //3b initialize another state that will hold the data for the first index from the city name array
+
+
+
+//4. define a side effect which will run ONCE on component mount to make a request to the API
+// city names
+
+const callCity = (city) => {
+    axios(
+        {
+            baseURL: 'https://dataservice.accuweather.com/locations/v1/cities/autocomplete',
+                params: {
+                    apikey: '5C7XkvLTaPsO5dFsQqLGb9Qffqwg114a',
+                    q: city
+                
+            }
+
+        }).then((apiData) => {
+            const info = apiData.data.filter(el => el.Country.ID === "CA" && el.AdministrativeArea.ID === "ON");
+            setCityKey(info[0].Key);
+           
+        })
+}
+
+
+
+
+
+const handleSubmit = (e) => {
+    e.preventDefault();
+    axios(
+        {
+            baseURL: `https://dataservice.accuweather.com/forecasts/v1/daily/1day/${cityKey}`,
+                params: {
```

Comment:
This axios call looks good! Check indentation to keep it consistent (i.e. probably no need for the extra indent for `params` vs `baseURL`; not a big deal in this case but if the object had more keys it could get harder to read)